Polygon.Net.Tests/FunctionalTests/NewsTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Polygon.Net.Models;
+using static Polygon.Net.Tests.TestManager;
+
+namespace Polygon.Net.Tests.FunctionalTests
+{
+    [TestClass]
+    public class NewsApiTests
+    {
+        private const string STATUS_OK = "OK";
+
+        [TestMethod]
+        public async Task GetNewsSucceedsAsync()
```

Comment:
Add some tests for the optional parameters you have in GetNewsAsync() (dates, tickers, etc). Make sure that not passing a ticker in returns news for all tickers and that passing a ticker in results in only news for that ticker. Same for dates, limit,  and sort.

Reply:
Fixed it